client/client.cpp

```diff
@@ -1,23 +1,78 @@
+#include <iostream>
+#include <sodium.h>
+#include <thrift/protocol/TBinaryProtocol.h>
+#include <thrift/transport/TSocket.h>
+#include <thrift/transport/TTransportUtils.h>
+
+#include "../constants/network.h"
 #include "../crypto/encryption_engine.h"
+#include "../gen-cpp/Operation_types.h"
+#include "../gen-cpp/RPC.h"
 #include "../host/redis.h"
-#include <iostream>
-using namespace std;
+
+using namespace apache::thrift;
+using namespace apache::thrift::protocol;
+using namespace apache::thrift::transport;
+
+Operation genRandOperation() {
+	float r = (float) rand() / RAND_MAX;
+	int key = rand() % KEY_MAX;
+  
+	Operation op;
+    
+	op.__set_op(r < 0.5 ? "PUT" : "GET");
+  op.__set_key(std::string(std::to_string(key)));
+	if(op.op == "GET") {
+		char value[VALUE_SIZE];
+		randombytes_buf(value, VALUE_SIZE);
+		op.__set_value(std::string(value));
+	}
+	return op;
+}
+
+void client(std::vector <float>* latencies) {
+	std::shared_ptr<TTransport> socket(new TSocket(HOST_IP, HOST_PORT));
+	std::shared_ptr<TTransport> transport(new TBufferedTransport(socket));
+	std::shared_ptr<TProtocol> protocol(new TBinaryProtocol(transport));
+	RPCClient client(protocol);
+	Operation op;
+
+	try {
+		transport->open();
+
+		for (int i = 0; i < 10; ++i) {
+			op = genRandOperation();
+			std::string val;
+			client.access(val, op);
+		}
+		
+		transport->close();
+	} catch (TException& tx) {
+    std::cout << "ERROR: " << tx.what() << std::endl;
+  }
+}
```

Comment:
(discussion) I'm going to add a clang-format check to enforce some formatting standards, this indentation is absolutely crazy (probably not on you)

Reply:
Yep that'd be great to have